pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerImpl.java

```diff
@@ -1337,15 +1337,15 @@ protected static final class OpSendMsg {
         SendCallback callback;
         Runnable rePopulate;
         ChunkedMessageCtx chunkedMessageCtx;
-        long uncompressedSize;
-        long sequenceId;
-        long createdAt;
-        long firstSentAt;
-        long lastSentAt;
-        int retryCount;
```

Comment:
Does it have any effect if no explicit initial value ?

Reply:
Yes. These initial values are all relied upon by other codes. Such as here:
https://github.com/apache/pulsar/blob/master/pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerImpl.java#L1409-L1422